### DISCUSSION TITLE
Not getting CGM readings on GC build

### DISCUSSION THREAD

**Discussion by kilovictor29745** [NightscoutFoundation/xDrip#2484] [Q&A]
My wife is a looper, Dexcom G6.

We've been using Nightscout from the beginning of this journey and this is the first time I'm having to ask, what I imagine, will be a pretty stupid question.

We're currently set up on heroku/atlas.  I ran through the GC installation (twice now).  

I seem to be missing the step that tells the CGM to send the readings to NS, or NS to go get the CGM readings.  I know it's something stupid, but i'm unable to see it.  Her CGM/loop device is an iphone dedicated to the task.

I'm able to get loop to send all but the CGM readings to NS.

We are appreciative of all of the community's efforts!

Kilo

EDIT:

I realize that the BRIDGE config vars are likely the issue.  I got those into the new NS build ( password, server and username) from the heroku config vars and rebooted.  Still no joy.

**Comment by Navid200** [comment]
So, you can see the site in a web browser, right?

Did you see this page?
https://navid200.github.io/xDrip/docs/Nightscout/NS_setup
At the bottom, it explains how to set up xDrip to upload.  You should be able to set other similar apps to upload the same way.

**Reply by kilovictor29745** [reply]
I appreciate the response!

I did see that, though that doesn't apply, as we don't have a NS master phone to put those settings into.

I did try to use loop to do this, but it only uploaded the loop related info, not the BS readings.

**Reply by Navid200** [reply]
I don't understand.
What do you expect to upload CGM readings to Nightscout?
Whatever it is, needs authority to upload.  You should be able to authorize it using the API-SECRET.

**Reply by kilovictor29745** [reply]
I appreciate your patience.

I went through and changed all of the config var settings to the ones that were in the heroku config vars.

The result is, its reading the dexcom clarity data.  The downside is it's pointing to the heroku set mongo/atlas DB.

Config var MONGO_CONNECTION
mongodb+srv://heroku_******@cluster-******1nl.mongodb.net/heroku_********=true&w=majority

I would assume that we would want this point to the GCP VM?

**Reply by Navid200** [reply]
Please delete the identifying codes/passwords/IDs from your message.

**Reply by kilovictor29745** [reply]
done.     i think...    :)
